api/v1/awsloadbalancercontroller_types.go

```diff
@@ -116,11 +116,19 @@ type AWSLoadBalancerControllerSpec struct {
 	// The secret is required to be in the operator namespace.
 	// If this field is empty, the credentials will be
 	// requested using the Cloud Credentials API,
-	// see https://docs.openshift.com/container-platform/4.11/authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.html.
+	// see https://docs.openshift.com/container-platform/4.13/authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.html.
 	//
 	// +kubebuilder:validation:Optional
 	// +optional
 	Credentials *configv1.SecretNameReference `json:"credentials,omitempty"`
+
+	// credentialsRequestConfig specifies further customization options for the controller's CredentialsRequest.
+	// This field won't have any effect if credentials have already been provided through the `Credentials` field,
+	// as a request for credentials from the Cloud Credentials Operator will not be triggered.
+	//
+	// +kubebuilder:validation:Optional
+	// +optional
+	CredentialsRequestConfig *AWSLoadBalancerCredentialsRequestConfig `json:"credentialsRequestConfig,omitempty"`
```

Comment:
So this allows me to configure a credentials request that is created by the load balancer type? Or to refer to an existing credentials request?